web/renderer/components/pageComponents/ConnectionsPage/NewConnection/StartDoltServerForm.tsx

```diff
@@ -46,7 +46,7 @@ export default function StartDoltServerForm() {
           setState({ port: p });
           setErr(undefined);
         }}
-        placeholder="Enter port number"
+        placeholder="e.g. 1234 (required)"
```

Comment:
Makes more sense to use default mysql port I think
```suggestion
        placeholder="e.g. 3306 (required)"
```